YumemiTraining/YumemiTraining/WeatherViewController.swift

```diff
@@ -46,30 +47,43 @@ final class WeatherViewController: UIViewController {
     
     private func loadWeather() {
         
-        do {
-            // WeatherPrameterを作成
-            let weatherPrameter = WeatherParameter(area: "tokyo", date: "2020-04-01T12:00:00+09:00")
-            
-            // 天気予報をAPIから取得
-            let weatherResult = try WeatherAPI.fetchWeather(weatherPrameter)
-            
-            // 天気の画像を設定
-            let weatherImageResource = self.weatherImageResource(weatherResult.weather)
-            self.weatherImageView.image = weatherImageResource.image
-            self.weatherImageView.tintColor = weatherImageResource.color
-            
-            //最高気温と最低気温を設定
-            self.minTempLabel.text = String(weatherResult.minTemp)
-            self.maxTempLabel.text = String(weatherResult.maxTemp)
-            
-        } catch YumemiWeatherError.invalidParameterError {
-            self.showErrorAlert(title: "天気情報の取得に失敗", message: "invalidParameterError")
-        } catch YumemiWeatherError.unknownError {
-            self.showErrorAlert(title: "天気情報の取得に失敗", message: "unknownError")
-        } catch {
-            print(error)
-        }
+        // weatherPrameterを作成
+        let weatherPrameter = WeatherParameter(area: "tokyo", date: "2020-04-01T12:00:00+09:00")
         
+        // 天気予報をAPIから取得
+        self.activityIndicatorView.startAnimating()
+        DispatchQueue.global().async {
+            let weatherResult = Result { try WeatherAPI.fetchWeather(weatherPrameter)}
```

Comment:
@torakyun 
`{ ... }` で囲うときは、内側にスペースを開けるようにしましょう。
```suggestion
            let weatherResult = Result { try WeatherAPI.fetchWeather(weatherPrameter) }
```

Reply:
https://github.com/torakyun/yumemi_training/pull/17/commits/ad5b5d29d0c1262092d719a767d85c558d7645b1
このコミットで修正しました。